### ISSUE TITLE
Running Linux on Dev machine

### ISSUE THREAD

**Issue by DannyDannyDanny** [DannyDannyDanny/methodology#5]
## First linux dev machine?

**TL;DR:** [Etch](https://www.balena.io/etcher/) [~kubuntu~](https://kubuntu.org/) [artix-cinnamon](https://artixlinux.org/download.php) onto a USB stick and install it.

![xkcd comic on linux](https://imgs.xkcd.com/comics/cautionary.png)

<details>

<summary>
<b>FAQ:</b> What linux should I run?

</summary>

...is like asking what instrument should I play? It mostly depends on what music you want to play.

_Distro-hopping_: Try some different linux distributions and see what you like.

After hopping for a bit, you'll notice there's more under the hood than _which linux to distro to run_. Most variants startup, behave or even looks somewhat similar. That's because on top of the base distribution there are many many "OS components". So the linux distro doesn't really matter. The real question is: _what components to run?_

</details>

<details>

<summary>
<b>FAQ:</b> What components should I run?

</summary>

The components are not well defined. Most operating system have a _Kernel_, some _Utilities_, maybe a _Graphics Server_, maybe a _Desktop Environment_, maybe a _Window Manager_ and bunch of associated applications.

![image](https://user-images.githubusercontent.com/16453674/156000489-517b841a-d915-4d5b-bdca-2a7a868f0804.png)

Defining the different component categories is non-trivial. Therefore you'll find different component-definitions in the image above, in [this system configuration](https://github.com/Unathi-Skosana/.files#system-configuration) and in the images below.

![image](https://user-images.githubusercontent.com/16453674/156384774-bb8cb5d5-be77-47b1-9ae2-7db40010fd17.png)

I mean look at this:

![image](https://user-images.githubusercontent.com/16453674/156386059-5e368feb-03cb-4abc-a49a-251cce016ddc.png)

</details>

## Conclusion 1

If you have a lot of spare time follow a guide, otherwise use (k)ubuntu to start with.

**Comment by DannyDannyDanny** [comment]
* **OS Kernel:** Linux
* **Utilities:** GNU
*

**Comment by DannyDannyDanny** [comment]
[Luke Smith](https://github.com/lukesmithxyz) seems to know what he's on about, here's a good starting point:

* [Luke Smith 1a](https://www.youtube.com/watch?v=4PBqpX0_UOc)
* [Luke Smith 1b](https://www.youtube.com/watch?v=nSHOb8YU9Gw)
* [Luke Smith 2](https://www.youtube.com/watch?v=4PBqpX0_UOc)

**Comment by DannyDannyDanny** [comment]
## Install LARBS

* [Github](https://github.com/LukeSmithxyz/LARBS)
* [lukesmith.xyz](https://larbs.xyz/)

## Misc
* Remeber to add all this to [setup-dev-machine.md](https://github.com/DannyDannyDanny/methodology/blob/main/setup-dev-machine.md)

**Comment by DannyDannyDanny** [comment]
### Requirement Testing Devices
* [MacBook Air](https://everymac.com/systems/apple/macbook-air/specs/macbook-air-core-i7-1.8-11-mid-2011-specs.html)
* Lenovo Thinkpad E540

### Requirement Tests
* 🚧 Must be able to achieve [#keyboard-layout](#issuecomment-1066734583) mapping
  * 🚧  what about function keys?
* 🚧 Must have nice touchpad driver (how to test? Manjaro Sway scroll friction is far too low)

**Comment by DannyDannyDanny** [comment]
## Prog.csv

If you're making your own installer, you're gonna need your own [progs.csv](https://github.com/LukeSmithxyz/LARBS/blob/master/progs.csv)

**Comment by DannyDannyDanny** [comment]
# Run ubuntu
